gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -174,35 +169,13 @@ private void computeTimestampForExportCorner(TTimestampWrapper timestampWrapper,
             BigDecimal miecValue = timestampWrapper.getMibiecValue().subtract(timestampWrapper.getAntcfinalValue());
             tcDocumentTypeWriter.fillTimestampExportCornerSuccess(timestampWrapper.getTimestamp(), miecValue);
         } else {
-            int italianImportAfterCep70Adjustment = timestampWrapper.getMiecIntValue();
-            int maxItalianSecureImport = timestampWrapper.getMibiecIntValue() - timestampWrapper.getAntcfinalIntValue();
-            String timeValue = timestampWrapper.getTimeValue();
-            String referenceCalculationTimeValue = timestampWrapper.getReferenceCalculationTimeValue();
-            TLimitingElement limitingElement = timestamp.getLimitingElement();
-            TTTCLimitedBy ttcLimitedBy = timestamp.getTTCLimitedBy();
-            TextType cgmFile = timestamp.getCGMfile();
-            TextType gskFile = timestamp.getGSKfile();
-            TextType cracFile = timestamp.getCRACfile();
-            List<TCalculationDirection> calculationDirections = timestamp.getCalculationDirections().get(0).getCalculationDirection();
-            List<AreaType> inArea = calculationDirections.stream().map(TCalculationDirection::getInArea).filter(at -> !eicCodesConfiguration.getItaly().equals(at.getV())).collect(Collectors.toList());
-            List<AreaType> outArea = calculationDirections.stream().map(TCalculationDirection::getOutArea).filter(at -> !eicCodesConfiguration.getItaly().equals(at.getV())).collect(Collectors.toList());
-            Map<String, QuantityType> shiftingFactorsMap = timestamp.getShiftingFactors().getShiftingFactor().stream()
-                    .collect(Collectors.toMap(f -> f.getCountry().getV(), TFactor::getFactor));
-
-            throw new NotImplementedException("Export corner handling is not implemented yet. "
-                    + "Italian import after CEP70 : " + italianImportAfterCep70Adjustment
-                    + " ; Max italian secure import : " + maxItalianSecureImport
-                    + " ; Time : " + timeValue
-                    + " ; Reference calculation time : " + referenceCalculationTimeValue
-                    + " ; Limiting element : " + limitingElement
-                    + " ; TTC limited by : " + ttcLimitedBy
-                    + " ; CGM file : " + cgmFile
-                    + " ; GSK file : " + gskFile
-                    + " ; CRAC file : " + cracFile
-                    + " ; InArea : " + inArea
-                    + " ; OutArea : " + outArea
-                    + " ; Shifting factors : " + shiftingFactorsMap
-            );
+            if (checkFranceInArea(timestamp)) {
```

Comment:
Same remark as in CseValidRequestValidator on line 60 : "if (checkFranceInArea)" is not easy to read and understand straight away. I consider "if (isFranceInArea())" as it is closer to what you would say in a real conversation.